eSchool.sln

```diff
@@ -27,19 +27,34 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebStatus", "src\Web\WebSta
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Libraries", "Libraries", "{74511F4E-FF9D-42C4-9531-A75C61270B73}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry", "src\Libraries\OpenTelemetry\OpenTelemetry.csproj", "{7B410F3B-36E0-4853-9B4E-41D0CC2865B5}"

```

Comment:
Please ensure no unintended changes are pushed. You don't have to change id of project you are not working with. If your IDE do this. please revert back to the actual value

Reply:
I don't think. I have changed it. I already have in the upstream.  Please visit the link. https://github.com/OpenCodeFoundation/eSchool/blob/master/eSchool.sln